src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/TrumpetSchwarzschild.hpp

```diff
@@ -32,14 +32,48 @@ namespace gr::Solutions {
  *
  * \details
  * This solution is a trumpet Schwarzschild black hole in the isotropic
- * coordinates. It is time-independent puncture solution in 1+log slicing.
+ * coordinates. It is a time-independent puncture solution in 1+log slicing.
  * The solution cannot be written down analytically in Schwarzschild
  * coordinates or isotropic coordinates. Refer to \cite Hannam2008
  * for equations and details.
  *
+ * We first set up a source grid in isotropic radial coordinate r, on which
+ * we compute the lapse and Schwarzschild radial coordinate R. The lapse
+ * is computed by solving eq. (54)-(56) using the toms748 algorithm, where
+ * the integrals in eq. (54)-(56) are evaluated by a tanh_sinh integrator.
+ * A standard Gauss quadratures will fail due to singular
+ * behaviors of the integrands near the integration limit. The Schwarzschild R
+ * is then computed by solving eq. (39) using again the toms748 algorithm. Note
+ * that for a value of n near 2. (corresponding to standard 1+log slicing),
+ * eq. (39) has two positive roots for a fixed lapse in [0., 1.). We choose
```

Comment:
[optional] Maybe include your Mathematica plot as a png to show the two solutions?